recipes/hexl/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.13.0)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(KEEP_RPATHS)
```
keep better compatibility on OSX

---

Done